packages/compass-logging/src/index.ts

```diff
@@ -27,25 +22,30 @@ export function createLoggerAndTelemetry(component: string): {
   debug: ReturnType<typeof createDebug>;
   track: TrackFunction;
 } {
-  // This application may not be running in an Node.js/Electron context.
-  const ipc: HadronIpcRenderer | null = isElectronRenderer
```

Comment:
`hadron-ipc` does this check itself.

Reply:
`hadron-ipc` logs a warning when it’s being used outside of electron, though, and it always expects to run in a Node.js environment currently (accesses `process.env`).

Reply:
Reverted.